### PR TITLE
Qualcomm_12-27-2021

### PR DESCRIPTION
**Pull request by QC-CNA**

### CODE REVIEW

File: 2021/30xxx/CVE-2021-35093.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{
```
Comment (cve-team) [inline 2021/30xxx/CVE-2021-35093.json]: Expected location (2021/35xxx/CVE-2021-35093.json) of file does not match actual location (2021/30xxx/CVE-2021-35093.json)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by anthnysingleton** [comment]
@QC-CNA Please resolve the Auto Validation errors. Once resolved we can move forward and process the request.

**Comment by QC-CNA** [comment]
Validation errors resolved for CVE-2021-35093.json. Updated branch Qualcomm_12-27-2021
please let us know if anything else needs to be updated.

**Comment by QC-CNA** [comment]
Hello ,

Validation errors are resolved. Please let us know if any additional change required

Regards
Sameesh

**Pull request merged**